k8s/cloud-build.sh

```
@@ -1,11 +1,12 @@
 #!/bin/sh
 # first arg : name of image to build
-# second arg (optional) : working directory. If not provided, it defaults to "../$1"
+# second arg: path to the Dockerfile, relative to the resolved WORKING_DIR.
+# thid arg (optional) : working directory. If not provided, it defaults to "../$1"
 set -e
 
-
 SCRIPT_DIR="$(cd "$(dirname "$0")" >/dev/null 2>&1 && pwd)"
-WORKING_DIR="${2:-${SCRIPT_DIR}/../$1}"
+WORKING_DIR="${3:-${SCRIPT_DIR}/../$1}"
```

Why not just keep the new $3 ?

---

I would be in favor of removing the default so that it fails faster